corehq/apps/domain/tests/test_forms.py

```diff
@@ -89,54 +91,41 @@ def test_confirmation_link_expiry_custom_present_when_flag_set(self):
         self.assertEqual(100, self.domain.confirmation_link_expiry_time)
 
     def test_confirmation_link_expiry_error_when_invalid_value(self):
+        OperatorCallLimitSettings.objects.all().delete()
```

Comment:
Would it make more sense to put this in a setup or teardown function?

Reply:
It is already added to the teardown. This is added for this specific test as the setup is creating an entry in this table, which adds a validation error because related setting `call_limit` is not set. Didn't add the setting as it was not relevant to this testcase.